content/docs/self-hosted-runners.md

```diff
@@ -413,3 +413,46 @@ corresponding
 [GitHub](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#hardening-for-self-hosted-runners)
 and [GitLab](https://docs.gitlab.com/runner/security) documentation for
 additional guidance.
+
+## Debugging 
```

Comment:
kinda overlaps with the FAQs in /ref/runner - wdyt?

My general feeling:

- /ref/runner: low level reference (list of options & full explanation of their arguments)
- /self-hosted-runners: high-level guide (with only necessary low-level details required to be a self-contained tutorial)

Reply:
kewl, let's add a debuggin link at the bottom of `/ref/runner` linking to this page then :)